backend/src/main/java/com/github/clarakraus/susume/post/PostService.java

```diff
@@ -19,8 +19,8 @@ public class PostService {
 
     private final SusumeMapper susumeMapper;
 
```

Comment:
maybe you can use the db for checking if it already exists here too? something like existsByCreatorAndId